.env.example

```
@@ -0,0 +1,43 @@
+TELEGRAM_BOT_TOKEN=...         # Telegram bot token
+DISCORD_BOT_TOKEN=...          # Discord bot token  
+SLACK_BOT_TOKEN=xoxb-...       # Slack bot token
+SLACK_APP_TOKEN=xapp-...       # Slack app token
+WHATSAPP_ACCESS_TOKEN=...      # WhatsApp bot access token
+WHATSAPP_PHONE_NUMBER_ID=...   # WhatsApp phone number ID
+
+# === Optional: Database ===
+DATABASE_URL=sqlite:///~/.praison/database.sqlite  # Database connection string
+
+# === Optional: Development ===
+LOG_level=INFO                 # Logging level (DEBUG, INFO, WARNING, ERROR)
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The LOG_level key should be in uppercase

(LowercaseKey)